src/PostgREST/QueryBuilder.hs

```diff
@@ -418,7 +418,7 @@ pgFmtFilter table (Filter fld (Operation hasNot_ ex)) = notOp <> " " <> case ex
    (op, VText val) -> pgFmtFieldOp op <> " " <> case op of
      "like"  -> unknownLiteral (T.map star val)
      "ilike" -> unknownLiteral (T.map star val)
-     "@@"    -> "to_tsquery(" <> unknownLiteral val <> ") "
```

Comment:
i am sorry, you are right, i was duped by the renaming of the test title.
looking at why this was not detected